l/LÖVE.lua

```diff
@@ -0,0 +1,3 @@
+function love.draw()
+    love.graphics.print("Hello World", 400, 300)
```

Comment:
Would it be better to draw the text at the origin?
```suggestion
    love.graphics.print("Hello World", 0, 0)
```

Reply:
Running this code will cause an 800 by 600 window to appear, and display white text (`hello world`) on a black background. 
Since doubling `0` will be `0`, the window will not display.